Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 406)
+++ CMakeLists.txt	(working copy)
@@ -47,14 +47,14 @@
              src/utility.c)
 
 # Set standard libraries
-SET (LIBS python2.4
+SET (LIBS python2.5
           dbus-1
           polkit-dbus
           db)
 
 # Include standard headers
 INCLUDE_DIRECTORIES (include/
-                     /usr/include/python2.4
+                     /usr/include/python2.5
                      /usr/include/dbus-1.0
                      /usr/lib/dbus-1.0/include
                      /usr/include/PolicyKit)
